dask/array/linalg.py

```diff
@@ -0,0 +1,125 @@
+"""
+   Copyright (c) 2015, Mariano Tepper, Duke University.
+   All rights reserved.
+
+   This file is part of dask and is under the BSD 3-Clause License,
+   which can be found at http://opensource.org/licenses/BSD-3-Clause
+"""
```

Comment:
I always hesitate about these author headers and the "all rights reserved" message.  Invariably others will edit this code and keeping track of authors in the source file itself quickly becomes difficult.  I'm happy to provide attribution in other places.  Perhaps there would be a good place in the docs to describe particular contributions?  This would be both more visible and less tied to the current state of code.

Also, you don't need to include the reference to the dask license.  My understanding is that the `LICENSE` file in the root of the directory applies to all code contained here.